stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
@demaledetti is the `stack.yaml.lock` file generate from stack?
If it is that the case, should not be better not include in the repo?

Reply:
For the record, we decided to go with this:
https://groups.google.com/forum/m/#!topic/haskell-stack/xeTN_3UNwE4